components/provisioner/internal/provisioning/input_converter.go

```diff
@@ -107,6 +111,7 @@ func (c converter) gardenerConfigFromInput(runtimeID string, input *gqlschema.Ga
 		LicenceType:                         input.LicenceType,
 		EnableKubernetesVersionAutoUpdate:   util.UnwrapBoolOrDefault(input.EnableKubernetesVersionAutoUpdate, c.defaultEnableKubernetesVersionAutoUpdate),
 		EnableMachineImageVersionAutoUpdate: util.UnwrapBoolOrDefault(input.EnableMachineImageVersionAutoUpdate, c.defaultEnableMachineImageVersionAutoUpdate),
+		AllowPrivilegedContainers:           util.UnwrapBoolOrDefault(input.AllowPrivilegedContainers, c.shouldAllowPrivilegedContainers(tillerYaml)),
```

Comment:
I wonder if we do not want some mechanism to override this field for example by Env var. As in the case, the 1.15 without the Tiller will not work correctly with disallowed privileged containers the Provisioner will end up blocking the release and the code changes will be needed to unblock it.

I would suggest adding something temporarily to prevent such a scenario as well as add follow up issue for cleaning it up and testing when `allowPrivilegedContainers` if set to false.

Reply:
It makes sense. But there will be three sources of this value then - the environment variable, the graphql mutation field, and the shouldAllowPrivilegedContainers mechanism
What priorities do you suggest? At least two of them won't be nil because the env variable will always have a value and the shouldAllowPrivilegedContainers mechanism will always return a value